### ISSUE TITLE
error when run it on local pc "cannot find bundle.js"

### ISSUE THREAD

**Issue by dolfijn3000** [naturalcrit/homebrewery#594]
i have node.js and npm experience but i cant get this to work localy always when the sever runs and i want to join in browser i get the error "cannot find bundle.js"

how to fix this ?

**Comment by HeruAsatru** [comment]
The currently availabe version of the homebrewery doesnt run out of the box anymore. Check out https://github.com/stolksdorf/homebrewery/issues/442 for a possible solution.

**Comment by dolfijn3000** [comment]
it almost worked now it has created the build folder but its still missing bundel.js

**Comment by HeruAsatru** [comment]
Hmm there is no bundle.js in the currently available repository. I can't look into the one I'm running at the moment. Is "cannot find bundle.js" the only thing the error says? or is there maybe a little more information?

**Comment by dolfijn3000** [comment]
There is other information but that error is the only thing that is colored

**Comment by HeruAsatru** [comment]
hmm strange. I don't have a bundle.js in my homebrewery and it works ... i really dont know what to say. have you seen where the bundle.js is called? maybe there's a path somewhere around that location.

**Comment by dolfijn3000** [comment]
so i got a bundle.js from an older version but now i get another error here is my whole output:

>  homebrewery@2.7.4 start D:\npm-projects\homebrewery
>  node server.js
> 
> server on port:8000
> (node:17320) DeprecationWarning: `open()` is deprecated in mongoose >= 4.11.0, use `openUri()` instead, or set the `useMongoClient` option if using `connect()` or `createConnection()`. See http://mongoosejs.com/docs/connections.html#use-mongo-client
> object
> 
> Error: Component was improperly built. Check the /build folder.
>   D (\npm-projects\homebrewery\node_modules\vitreum\steps\render.js:45)
> 
> Error: Component was improperly built. Check the /build folder.
>     at getBody (D:\npm-projects\homebrewery\node_modules\vitreum\steps\render.js:45:57)
>     at err (D:\npm-projects\homebrewery\node_modules\vitreum\steps\render.js:57:17)
>     at render (D:\npm-projects\homebrewery\node_modules\vitreum\steps\render.js:55:9)
>     at app.use (D:\npm-projects\homebrewery\server.js:119:2)
>     at Layer.handle [as handle_request] (D:\npm-projects\homebrewery\node_modules\express\lib\router\layer.js:95:5)
>     at trim_prefix (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:317:13)
>     at D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:284:7
>     at Function.process_params (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:335:12)
>     at next (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:275:10)
>     at D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:635:15
>     at next (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:260:14)
>     at Function.handle (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:174:3)
>     at router (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:47:12)
>     at Layer.handle [as handle_request] (D:\npm-projects\homebrewery\node_modules\express\lib\router\layer.js:95:5)
>     at trim_prefix (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:317:13)
>     at D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:284:7
> object
> 
> Error: Component was improperly built. Check the /build folder.
>   D (\npm-projects\homebrewery\node_modules\vitreum\steps\render.js:45)
> 
> Error: Component was improperly built. Check the /build folder.
>     at getBody (D:\npm-projects\homebrewery\node_modules\vitreum\steps\render.js:45:57)
>     at err (D:\npm-projects\homebrewery\node_modules\vitreum\steps\render.js:57:17)
>     at render (D:\npm-projects\homebrewery\node_modules\vitreum\steps\render.js:55:9)
>     at app.use (D:\npm-projects\homebrewery\server.js:119:2)
>     at Layer.handle [as handle_request] (D:\npm-projects\homebrewery\node_modules\express\lib\router\layer.js:95:5)
>     at trim_prefix (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:317:13)
>     at D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:284:7
>     at Function.process_params (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:335:12)
>     at next (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:275:10)
>     at D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:635:15
>     at next (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:260:14)
>     at Function.handle (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:174:3)
>     at router (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:47:12)
>     at Layer.handle [as handle_request] (D:\npm-projects\homebrewery\node_modules\express\lib\router\layer.js:95:5)
>     at trim_prefix (D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:317:13)
>     at D:\npm-projects\homebrewery\node_modules\express\lib\router\index.js:284:7

**Comment by LimpingNinja** [comment]
Same issue

**Comment by LimpingNinja** [comment]
@AlexeySachkov Hello Alexey, those instructions were followed exactly. I'm thinking my issue wasn't identical to this posters, with mine being related to vitreum errors during install and build phases, with the end result being failure. I tried a few tweaks myself and the end result was a running version that served no connections. No set up building based on the 'getting started' worked.

Ultimately I followed Spencer Baynton's post in this issue #442 to resolve the issues building and running Homebrewery: 

Whether that was the correct solution or not is up for debate, I will say that after doing that it does indeed run.
